s/elements/stick/element.ts

```diff
@@ -21,24 +22,23 @@ export class NubStick extends MagicElement {
 	realize() {
 		const {use} = this
 
-		const [, setTrackingMouse, getTrackingMouse] =
-			use.state(false)
-
-		const [, setTrackingTouchId, getTrackingTouchId] =
+		const [, setTrackingPointerId, getTrackingPointerId] =
 			use.state<number | undefined>(undefined)
 
-		const [styleTransforms, setStyleTransforms] =
-			use.state({stick: "", understick: ""})
+		const [vector, setVector] = use.state({
+			x: 0,
+			y: 0
+		})
```

Comment:
let's use the V2 type for all 2d vectors (so it's `[1, 2]` instead of `{x: 1, y: 2}`)